### PR TITLE
Feature/tenant config

### PR DESCRIPTION
**Pull request by tejassonar**

### CODE REVIEW

File: public/tenants/treesforjane/config.ts [hunk 1]
```diff
@@ -1,81 +1,64 @@
 const config = {
-  // name of tenant
-  tenantName: 'treesforjane',
-  // url of tenant home page
-  tenantURL: 'https://treesforjane.org/',
+  tenantName: 'janegoodall',
```
Comment (sagararyal) [inline public/tenants/treesforjane/config.ts:2]: this should be treesforjane

File: public/tenants/treesforjane/config.ts [hunk 2]
```diff
@@ -1,81 +1,64 @@
...
+  tenantURL: 'map.treesforjane.org',
+  languages: ['de', 'en'],
```
Comment (sagararyal) [inline public/tenants/treesforjane/config.ts:4]: en only for now.

File: public/tenants/treesforjane/config.ts [hunk 3]
```diff
@@ -1,81 +1,64 @@
...
+    image: `${process.env.CDN_URL}/media/images/app/bg_layer.jpg`,
+    twitterHandle: '',
+    locale: 'en_US',
+  },
+  home: {
+    descriptionTitle: '',
+    image: `${process.env.CDN_URL}/media/images/app/bg_layer.jpg`,
   },
-  languages: ['en'],
-  hideNavbar: true,
```
Comment (sagararyal) [inline public/tenants/treesforjane/config.ts:21]: how does navbar work with register trees?

**Comment by norbertschuler** [comment]
What the purpose of this PR as treesforjane.org is already deployed?

**Comment by tejassonar** [comment]
> What the purpose of this PR as treesforjane.org is already deployed?

Yes. we can delete this branch now.

**Pull request closed without merging**